pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/prometheus/role_binding.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package prometheus
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+)
+
+var _ machinery.Template = &RoleBinding{}
+
+// RoleBinding scaffolds a file that defines the prometheus service monitor and role binding
+type RoleBinding struct {
+	machinery.TemplateMixin
+}
+
+// SetTemplateDefaults implements file.Template
+func (f *RoleBinding) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = filepath.Join("config", "prometheus", "role_binding.yaml")
+	}
+
```

Comment:
@machadovilaca, 

That is perfect. So, could we please update the metrics doc with the info? Example (something like):

<aside class="note">

Note that you need to ensure that the RBAC roles scaffold in the config/prometheus were also applied to the namespace where the manager is deployed. Otherwise, the [Prometheus operator]() will be unable to have access to the ServiceMonitor that will export the metrics. More info about the RBAC requirements when we are using the Prometheus Operator to have a Prometheus instance available and installed can be found in its [doc](link)

 </aside>

Reply:
added